spec/lib/journaled/audit_log_spec.rb

```diff
@@ -78,14 +80,23 @@ def encrypted_attribute?(_key)
 
   describe '.excluded_classes' do
     let(:defaults) do
-      %w(
-        Delayed::Job
-        PaperTrail::Version
-        ActiveStorage::Attachment
-        ActiveStorage::Blob
-        ActiveRecord::InternalMetadata
-        ActiveRecord::SchemaMigration
-      )
+      if Gem::Version.new(Rails.version) < Gem::Version.new('7.1')
+        %w(
+          Delayed::Job
+          PaperTrail::Version
+          ActiveStorage::Attachment
+          ActiveStorage::Blob
+          ActiveRecord::InternalMetadata
+          ActiveRecord::SchemaMigration
+        )
+      else
+        %w(
+          Delayed::Job
+          PaperTrail::Version
+          ActiveStorage::Attachment
+          ActiveStorage::Blob
+        )
+      end
```

Comment:
I decided to just have the same if-expr here instead of fiddling with mocking the version and whatnot.